doc/source/programs/gdal_raster_polygonize.rst

```diff
@@ -0,0 +1,94 @@
+.. _gdal_raster_polygonize_subcommand:
+
+================================================================================
+"gdal raster polygonize" sub-command
+================================================================================
+
+.. versionadded:: 3.11
+
+.. only:: html
+
+    Create a polygon feature dataset from a raster dataset
+
+.. Index:: gdal raster polygonize
+
+Synopsis
+--------
+
+.. program-output:: gdal raster polygonize --help-doc
+
+Description
+-----------
+
+:program:`gdal raster polygonize` creates vector polygons for all connected
+regions of pixels in the raster sharing a common pixel value.
+Each polygon is created with an attribute indicating the pixel value of that polygon. The mask
+(either explicit, or implicit through nodata value or alpha band) associated
+to the selected band will be used to determine the validity of pixels.
+
+The utility can create the output vector datasource if it does not already exist,
+otherwise it may append to an existing one.
+
+The utility is based on the ::cpp:func:`GDALPolygonize` function which has additional
+details on the algorithm.
+
+The following options are available:
+
+Standard options
+++++++++++++++++
+
+
+.. include:: gdal_options/of_vector.rst
+
+.. include:: gdal_options/co.rst
+
+.. include:: options/lco.rst
+
+.. include:: gdal_options/overwrite.rst
+
+.. option:: --update
+
+    Whether the output dataset must be opened in update mode. Implies that
+    it already exists.
+
+.. option:: --overwrite-layer
+
+    Whether overwriting the existing output vector layer is allowed.
+
+.. option:: --append
+
+    Whether appending features to the existing output vector layer is allowed
+
+.. option:: -b, --band <BAND>
+
+    Picks a particular band to get the DEM from. Defaults to band 1.
```

Comment:
Picks a particular band to polygonize. Defaults to band 1.

Reply:
fixed